### ISSUE TITLE
npm warning: sprintf package is deprecated in favor of sprintf-js

### ISSUE THREAD

**Issue by piLinux** [strapi/strapi#1801]
**OS:**
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic

**Environment config:**

`nodejs -v`
v10.9.0

`npm -v`
6.2.0

`mongod --version`
db version v4.0.1
git version: 54f1582fc6eb01de4d4c42f26fc133e623f065fb
OpenSSL version: OpenSSL 1.1.0g  2 Nov 2017
allocator: tcmalloc
modules: none
build environment:
    distmod: ubuntu1804
    distarch: x86_64
    target_arch: x86_64

**Strapi using deprecated sprintf package:**

`npm install strapi@alpha -g`
npm WARN deprecated sprintf@0.1.5: The sprintf package is deprecated in favor of sprintf-js.

**Comment:**
Can you replace _sprintf_ package with _sprintf-js_ package? Thank you.

**Comment by lauriejim** [comment]
Hello @piLinux , thank you for this report.

It's an easy update to do, can you please submit a PR to fix it.
We will appreciate your contribution on Strapi.

Check out the contributing guide to get started: https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md

**Comment by avallete** [comment]
I've didn't found any migration guide for this deprecation. So it should not have any breaking changes.

If no one is already on it, I will investigate about the possibles side effect and submit a PR ASAP.

**Comment by avallete** [comment]
After some research, as I feared, `sprintf` is not a direct strapi dependency (as usual with node :sob: )

The error come from `koa-i18n` package, which depend on `i18n-2` module.

An issue and a PR is already ready on this repo. I've made a comment to bump up the issue.
https://github.com/jeresig/i18n-node-2/issues/120

Once the `i18n-2` PR has been merged, we will probably need to create an issue onto: https://github.com/koa-modules/i18n/issues

For strapi, the update should be automatic since `koa-i18n` version is marked as `^2.1.0`
